.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        node-version: [12.x]
+        node-version: [14.x]
```

Comment:
actually keep both 12 and 14 so we test both. That's the intent for the matrix. I'll add 14 to required checkecks.

Reply:
Done